PowerEditor/installer/nppSetup.nsi

```diff
@@ -172,7 +172,7 @@ updaterDone:
 
 	Call SetRoughEstimation		; This is rough estimation of files present in function copyCommonFiles
 	InitPluginsDir			; Initializes the plug-ins dir ($PLUGINSDIR) if not already initialized.
-	Call preventInstallInWin9x
+	Call preventInstall		; check unsupported OSes and CPUs

```

Comment:
Please rename `preventInstall` to `checkCompability`.

Reply:
Done.